CHANGELOG.md

```diff
@@ -1,4 +1,30 @@
 ## 3.40.0 (Unreleased)
+
+DEPRECATIONS:
+* bigtable: Deprecated `instance_type` for `google_bigtable_instance` - it is now recommended to leave field unspecified. ([#7253](https://github.com/hashicorp/terraform-provider-google/pull/7253))
+
+FEATURES:
+* **New Data Source:** compute: added `google_compute_region_ssl_certificate` datasource ([#7252](https://github.com/hashicorp/terraform-provider-google/pull/7252))
+* **New Resource:** `google_compute_target_grpc_proxy` ([#7277](https://github.com/hashicorp/terraform-provider-google/pull/7277))
+
+IMPROVEMENTS:
+* cloudfunctions: added the ALLOW_INTERNAL_AND_GCLB option to `ingress_settings` of `google_cloudfunctions_function` resource. ([#7287](https://github.com/hashicorp/terraform-provider-google/pull/7287))
+* cloudlbuild: added `options` and `artifacts` properties to `google_cloudbuild_trigger` ([#7280](https://github.com/hashicorp/terraform-provider-google/pull/7280))
+* compute: added GRPC as a valid value for `google_compute_backend_service.protocol` (and regional equivalent) ([#7254](https://github.com/hashicorp/terraform-provider-google/pull/7254))
+* compute: added support for configuring Internal load balancer for Cloud Run for Anthos ([#7268](https://github.com/hashicorp/terraform-provider-google/pull/7268))
+* compute: added 'all' option for `google_compute_firewall` ([#7225](https://github.com/hashicorp/terraform-provider-google/pull/7225))
+* dataflow : added `transformnameMapping` to `google_dataflow_job` ([#7259](https://github.com/hashicorp/terraform-provider-google/pull/7259))
+* dns: added `force_destroy` option to `google_dns_managed_zone` to delete records created outside of Terraform ([#7289](https://github.com/hashicorp/terraform-provider-google/pull/7289))
+* serviceusage: added ability to pass `google.project.id` to `project` property ([#7255](https://github.com/hashicorp/terraform-provider-google/pull/7255))
```

Comment:
even though there aren't a lot of options for serviceusage resources, it's probably still good to explicitly specify `google_project_service` here

Reply:
Done.